### PR TITLE
Please add Jakarta EE Specifications Meetings Calendar to the connect…

### LINKED ISSUE #1114: Please add Jakarta EE Specifications Meetings Calendar to the connect page
Can you please add a section on https://jakarta.ee/connect/ for Jakarta EE Specifications Meetings Calendar?

The section name can be *Jakarta EE Specifications Meetings Calendar*

Here is the link

https://calendar.google.com/calendar/embed?src=eclipse-foundation.org_e9ki8t2gc75sh07qdh95c8ofvc%40group.calendar.google.com&ctz=America%2FToronto

@karenmcnaughton please have a look and give your +1 as well.

This Calendar is open to the public if they are interested to attend any of the any of the specification project meetings.

### PR DESCRIPTION
**Pull request by autumnfound**
Closes #1114

### CODE REVIEW

**Review by chrisguindon — APPROVED** [review]

File: i18n/en.toml [hunk 1]
```diff
@@ -106,6 +106,18 @@
 [connect-blogs-link-text]
   other = "View"
 
+[connect-spec-calendar-title]
+  other = "Jakarta EE Specifications Meetings Calendar"
+
+[connect-spec-calendar-body]
+  other = "LOREM IPSUM."
```
Comment (ivargrimstad) [inline i18n/en.toml]: Change "LOREM IPSUM"  to "Public calendar for all Jakarta EE specification projects.

File: i18n/en.toml [hunk 2]
```diff
@@ -106,6 +106,18 @@
 [connect-blogs-link-text]
   other = "View"
 
+[connect-spec-calendar-title]
+  other = "Jakarta EE Specifications Meetings Calendar"
```
Comment (ivargrimstad) [inline i18n/en.toml]: Remove "Meetings", so the text is "Jakarta EE Specifications Calendar"

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
See comments.

**Review by ivargrimstad — COMMENTED** [review]
Can we group the spec calendar together with the community calendar? 
Or place it directly below?

**Comment by TanjaObradovic** [comment]
Maybe we can have a section Jakarta EE Calendars, and underneath Jakarta EE Community Calendar and  Jakarta EE Specifications Meetings Calendar; similar to what we have for Mailing Lists

**Comment by TanjaObradovic** [comment]
@autumnfound  do you want me to open another issue to group the Calendars? as per my comment above?

**Comment by autumnfound** [comment]
> @autumnfound do you want me to open another issue to group the Calendars? as per my comment above?

See https://deploy-preview-1117--jakartaee.netlify.app/connect/, I've made an update to lump them together!

**Comment by TanjaObradovic** [comment]
You are the best! thanks @autumnfound! Looks great!

The buttons are different under the Calendars, both can say "subscribe"

**Comment by autumnfound** [comment]
> You are the best! thanks @autumnfound! Looks great!
> 
> The buttons are different under the Calendars, both can say "subscribe"

@TanjaObradovic change is done, just waiting on reviews for this and then I can merge it!

**Review by ivargrimstad — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**